src/main/resources/db-scripts/cgds.sql

```diff
@@ -632,24 +631,6 @@ CREATE TABLE `copy_number_seg_file` (
   FOREIGN KEY (`CANCER_STUDY_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE
 );
 
--- --------------------------------------------------------
-CREATE TABLE `cosmic_mutation` (
-  `COSMIC_MUTATION_ID` varchar(30) NOT NULL,
-  `CHR` varchar(5),
-  `START_POSITION` bigint(20),
-  `REFERENCE_ALLELE` varchar(255),
-  `TUMOR_SEQ_ALLELE` varchar(255),
-  `STRAND` varchar(2),
-  `CODON_CHANGE` varchar(255),
-  `ENTREZ_GENE_ID` int(11) NOT NULL,
-  `PROTEIN_CHANGE` varchar(255) NOT NULL,
-  `COUNT` int(11) NOT NULL,
-  `KEYWORD` varchar(50) DEFAULT NULL,
-  KEY (`KEYWORD`),
-  PRIMARY KEY (`COSMIC_MUTATION_ID`),
-  FOREIGN KEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`)
-);
```

Comment:
Also since we are dropping an existing table, we should also updating db scheme version